backend/telemetry_core/src/feed_message.rs

```diff
@@ -201,6 +210,7 @@ impl FeedMessageWrite for AddedNode<'_> {
             node.block_details(),
             &node.location(),
             &node.startup_time(),
+            &hwbench,
```

Comment:
No particular reason, It felt a bit more appropriate to me since the `hwbench` is not part of the `NodeDetails`, but a separate field on the `Node` struct, similar to start `startup_time` and `location`.

Reply:
Okeydoke :)

Reply:
Have moved this to the `details` part of the serialization as `hwbench` is a logical part of the node details, although placed as a separate field on the node structure; and it may also help some UIs with deserializing over the wire message. Thanks!